nixpkgs_review/cli/__init__.py

```diff
@@ -164,6 +164,11 @@ def common_flags() -> List[CommonFlag]:
             action="store_true",
             help="Allow import-from-derivation",
         ),
+        CommonFlag(
+            "--allow-url-literals",
```

Comment:
I think I see a pattern here. I don't want to so many `--allow-$X` flags here. Instead a I would prefer a `--allow=ifd,aliases,url-literals` flag.

Reply:
Since this is mostly interactive usage I think we can safely drop existing flags without causing too much damage.

Reply:
If we have than some datastructure that can store all those flags than we don't need to pass down as many parameters to other functions.

Reply:
Sounds like a good idea, should I keep the --allow-aliases flag to show a deprecation message?

Reply:
Instead of comma separated, I opted for it beeing passed multiple times: `--allow ifd --allow aliases` since it is easier to implement with `choices`

Reply:
I think we can just drop the flag.